_posts/2020-07-06-whats-so-super-about-super-tdma.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+comments: true
+title: What's so "super" about Super-TDMA ? 
+date: 06/07/2020
+author: Prasad Anjangi
+categories: howto
+feature-img: "assets/img/stdma/stdma-unet.jpg"
+thumbnail: "assets/img/stdms/stdma-unet.jpg"
+tags: [howto, tdma, medium access control, long propagation delays, throughput, performance boost]
+---
+
+Underwater acoustic (UWA) networks will play a key role in many areas including marine, offshore and subsea industries in the future. There have been tremendous and impressive technological advances in underwater acoustic communications and networking field. The one that caught my attention and brought about a clear difference in the novel approaches that are needed to deal with challenges in UWA networks was the "exploitation" of large propagation delays that exist in UWA networks. 
```

Comment:
I would say UWA is already playing a key role now :) not in the future!

Reply:
Agreed :) Fixed now.